sqlobject/util/csvexport.py

```diff
@@ -168,7 +169,11 @@ def export_csv_zip(soClasses, file=None, zip=None, filename_prefix='',
         close_zip_when_finished = False
     else:
         return_when_finished = True
-        file = StringIO()
+        if sys.version_info[0] < 3:
+            file = StringIO()
```

Comment:
Why use a conditional here. Technically it's bytes on both versions and `BytesIO` will work on both versions. This should be simplified to not needing the conditional usage.